js-exercises/sleep-promise/sleepPromise.js

```diff
@@ -0,0 +1,18 @@
+const sleep = ms => {
+  if (typeof ms != 'number') {
+    throw new TypeError(`Expected Number, got ${typeof ms}, please try again with a number.`);
```

Comment:
Both the throw messages should be of same format such as 'Function expects a number as time value, got ${typeof ms}'.